requirements.txt

```diff
@@ -4,29 +4,32 @@
 # Format:
 # PackageName==X.Y.Z          # <comma separated list of WMCore software needing the package>
 
-Cheetah==2.4.0                # wmagent,reqmgr2,reqmon
-CherryPy==17.4.0              # wmagent,reqmgr2,reqmon,global-workqueue,reqmgr2ms
-CMSMonitoring>=0.3.4          # wmagent,reqmgr2,reqmon,reqmgr2ms
-coverage==4.5.4               # wmagent
-cx-Oracle==5.2.1              # wmagent
-dbs-client==3.13.1            # wmagent,reqmgr2,reqmon,global-workqueue
-decorator==3.4.2              # wmagent
-funcsigs==1.0.2               # wmagent
-future==0.18.2                # wmagent,reqmgr2,reqmon,global-workqueue,reqmgr2ms
-httplib2==0.19.0              # wmagent,reqmgr2,reqmon,acdcserver,global-workqueue,reqmgr2ms
-Markdown==3.0.1               # wmagent
-mock==2.0.0                   # wmagent
-MySQL-python==1.2.5           # wmagent
-nose2==0.9.2                  # wmagent
-psutil==5.6.6                 # wmagent,reqmgr2,reqmon,global-workqueue
-pycurl==7.43.0.3              # wmagent,reqmgr2,reqmon,global-workqueue,reqmgr2ms
-pymongo==3.10.1               # reqmgr2ms
-pyOpenSSL==18.0.0             # wmagent
-pyzmq==17.1.2                 # wmagent
-retry==0.9.1                  # wmagent,reqmgr2,reqmon,global-workqueue,reqmgr2ms
-rucio-clients==1.23.0         # wmagent,global-workqueue,reqmgr2ms
-setuptools==39.2.0            # wmagent
-Sphinx==1.3.5                 # wmagent,reqmgr2,acdcserver,global-workqueue,reqmgr2ms
-SQLAlchemy==1.3.3             # wmagent
-stomp.py==4.1.15              # wmagent
-CMSCouchapp==1.2.10           # wmagent
+Cheetah3~=3.2.6.post2         # wmagent,reqmgr2,reqmon
```

Comment:
Erik, is this list exactly the same one that we had in the requirements_py3.txt file? Can you please confirm whether the only differences are:
* a sorted list now
* and the version comparison specifier
?

Reply:
There are some additional changes:
* Removed `dbs3-pycurl` since it is a dependency of `dbs3-client`. 
* Removed `dbs3-client` as a dependency for wmagent-devtools since it was not in the [spec](https://github.com/cms-sw/cmsdist/blob/comp_gcc630/wmcorepy3-devtools.spec).
* Added `gfal2-python` for reqmgr2ms since it was in the [spec](https://github.com/cms-sw/cmsdist/blob/comp_gcc630/reqmgr2ms.spec)
* Added `mysqlclient` for wmagent since it was in the [spec](https://github.com/cms-sw/cmsdist/blob/comp_gcc630/py3-mysqlclient.spec) 

I will add these changes to the PR description as well.

Reply:
Wonderful! Thanks for catching those up!